workflows/common/python/compare.py

```diff
@@ -0,0 +1,162 @@
+import os
+import pandas as pd
+import pandas as pd
+import numpy as np
+from sklearn.metrics import mean_squared_error
+
+conditions = pd.DataFrame([[ 'nAromAtom' , 5, 10 ],
```

Comment:
Please add some context on what are we comparing and text on cancer drug discovery domain for this python file.